app/controllers/admin/loans_controller.rb

```diff
@@ -165,8 +165,12 @@ def prep_form_vars
   def prep_timeline
     filters = {}
     filters[:type] = params[:type] if params[:type].present?
+    filters[:status] = params[:status] if params[:status].present?
     @loan.root_timeline_entry.filters = filters
     @type_options = ProjectStep.step_type_option_set.translated_list
+    @status_options = ProjectStep::COMPLETION_STATUSES.map do |status|
+      [ I18n.t("project_step.completion_status.#{status}"), status ]
+    end.sort
```

Comment:
Nice. Though you should need to sort these. Just go with a sensible order in the constant.